caas/kubernetes/provider/bootstrap.go

```diff
@@ -679,6 +680,8 @@ func (c *controllerStack) ensureControllerConfigmapAgentConf() error {
 }
 
 func (c *controllerStack) ensureControllerServiceAccount() error {
+	ctx := context.TODO()
```

Comment:
`c.ctx` has a method to get the `context.Context()`
no need to create a new one here